synapsemonitor/monitor.py

```diff
@@ -124,7 +165,15 @@ def _find_modified_entities_container(syn: Synapse, syn_id: str, days: int = 1)
     Returns:
         List of synapse ids
     """
-    raise NotImplementedError("Projects and folders not supported yet")
+    syn_id_mod = []
+    syn_id_children = _traverse(syn, syn_id)
+
+    for syn_id_child in syn_id_children:
+        syn_id_res = _find_modified_entities_file(syn, syn_id_child, days)
+        if syn_id_res is not None:
```

Comment:
I don't think the function ever returns None.  I think it returns an empty list. This should be:
```suggestion
        if syn_id_res:
```

Reply:
Updated